src/plugins/data_views/server/rest_api_routes/public/scripted_fields/get_scripted_field.ts

```diff
@@ -49,7 +49,7 @@ export const registerGetScriptedFieldRoute = (
           response: {
             200: {
               body: schema.object({
-                field: serializedFieldFormatSchema,
+                field: schema.object(fieldSpecSchemaFields),
```

Comment:
Are these schemas identical?

Reply:
far from it! This is correcting bad code. Perhaps I should break it out into its own PR.

Reply:
I'd say if you're very confident it won't cause any schema validation issues, then it's ok to include. But if there's a risk, a separate PR would be better IMO so it's easier to trace back to if needed.

Reply:
So I guess this is the last remaining task to bring this over the finishing line, right?

Reply:
addressed as part of https://github.com/elastic/kibana/pull/168776